### PR TITLE
updates to pathology detail node DICT-225

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 1]
```diff
@@ -351,6 +381,8 @@ properties:
       - Supraclavicular
       - Unknown
       - Not Reported
+      - Aortic
+      - Pelvis, NOS
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:385]: it says "Pelvic" in the jira.

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 2]
```diff
@@ -325,6 +343,18 @@ properties:
...
+  lymph_node_dissection_method:
+    enum:
+      - Functional (Limited) Neck Dissection
+      - Modified Radical Neck Dissection
+      - Radical Neck Dissection
+
+  lymph_node_dissection_site:
+    enum:
+      - Neck, Left
+      - Neck,Right
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:355]: do we need an extra space between the comma and "R"? (jira is the same)

**Comment by wteouchicago** [comment]
overall (other than individual comments): just need some alphanumeric sort on enums.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 2 above)
Comment (bullenc) [reply gdcdictionary/schemas/pathology_detail.yaml:355]: copying error, thanks for point out. Fixed.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (bullenc) [reply gdcdictionary/schemas/pathology_detail.yaml:385]: Thanks, thought I had updated JIRA; when conferring with Megan Stine @ IMS, it was agreed enum should be "Pelvis, NOS".

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 3]
```diff
@@ -97,6 +97,8 @@ properties:
       - Tubular (papillary) adenoma(s)
       - Tumor-associated lymphoid proliferation
       - Tumor has rough spikey edges
+      - Bone marrow discordant histology
+      - Bone marrow concordant histology
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:101]: alpha sort :)

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 4]
```diff
@@ -270,6 +272,7 @@ properties:
       - Other
       - Unknown
       - Not Reported
+      - Esophageal Mucosa Columnar Dysplasia
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:275]: alpha sort

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 5]
```diff
@@ -290,6 +293,21 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  extracapsular_extension:
+    enum:
+      - Focal
+      - Extensive
+      - Not Reported
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:300]: alpha sort

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 6]
```diff
@@ -290,6 +293,21 @@ properties:
...
+
+  extranodal_extension:
+    enum:
+      - No Extranodal Extension
+      - Gross Extension
+      - Microscopic Extension
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:306]: alpha sort

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 7]
```diff
@@ -325,6 +343,18 @@ properties:
...
+    enum:
+      - Functional (Limited) Neck Dissection
+      - Modified Radical Neck Dissection
+      - Radical Neck Dissection
+
+  lymph_node_dissection_site:
+    enum:
+      - Neck, Left
+      - Neck, Right
+      - Neck, NOS
```
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:356]: no idea if we need to alpha sort this. does NOS  always come after the...non-NOSes?

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (wteouchicago) [inline gdcdictionary/schemas/pathology_detail.yaml:385]: alpha sort

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 7 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml:366]: Hah, I'm not sure it just looks correct, let's leave it.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml]: Fixed.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 6 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml]: Fixed.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 5 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml]: Fixed.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 4 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml]: Fixed.

File: gdcdictionary/schemas/pathology_detail.yaml (hunk 3 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml]: Fixed.

**Review by mpsolano — APPROVED** [review]

File: gdcdictionary/schemas/pathology_detail.yaml [hunk 8]
```diff
@@ -451,6 +493,14 @@ properties:
     type: integer
     minimum: 0
 
+  lymph_nodes_removed:
+    $ref:
+      - "_terms.yaml#/lymph_nodes_removed/common
```
Comment (mpsolano) [inline gdcdictionary/schemas/pathology_detail.yaml:498]: This should fix the build error:
```suggestion
      - "_terms.yaml#/lymph_nodes_removed/common"
```

Comment (wwysoc2) [reply gdcdictionary/schemas/pathology_detail.yaml]: Thanks @mpsolano!  Made the commit, didn't see that cool "Commit suggestion" feature until I was done.

**Review by bullenc — COMMENTED** [review]
file changes match JIRA ticket, alpha order looks good, looks good to me 👍

**Review by wteouchicago — APPROVED** [review]

**Review by tlicht3 — APPROVED** [review]

**Pull request merged**